core/domain/skill_services.py

```diff
@@ -727,6 +727,38 @@ def apply_change_list(skill_id, change_list, committer_id):
         raise e
 
 
+def populate_skill_model_with_skill(skill_model, skill):
```

Comment:
Are there ever cases where we would populate_X_model_with_Y where Y is not X?

If not, then I think the naming is a bit redundant and maybe we can call these functions something like populate_skill_model_fields()

Reply:
Yeah, it is always X = Y, done.